modules/create-wif-cluster-cli.adoc

```diff
@@ -186,6 +186,22 @@ $ ocm create cluster <cluster_name> \ <1>
 <12> Optional: Maximum number of compute nodes.
 <13> Optional: Secure Boot enables the use of Shielded VMs in the Google Cloud Platform.
 
+[id="ocm-cli-list-wif-commands_{context}"]
+== Listing WIF clusters
+
+To list all of your {product-title} clusters that have been deployed using the WIF authentication type, run the following command:
+
+[source,terminal]
+----
+$ ocm list clusters --parameter search="gcp.authentication.wif_config_id != ''"
+----
+To list all of your {product-title} clusters that have been deployed using a specific wif-config, run the following command:
+[source,terminal]
+----
+$ ocm list clusters --parameter search="gcp.authentication.wif_config_id = '<wif_config_id>'" <1>
+----
+<1> Replace `<wif_config_id>` with the ID of the WIF configuration to list the clusters that have been deployed using that WIF configuration.
```

Comment:
```suggestion
<1> Replace `<wif_config_id>` with the ID of the WIF configuration. 
```

Consider removing the repeated info "...list the clusters that have been deployed using that WIF..."